pkgs/applications/editors/neovim/neovim-gtk.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, rustPlatform
+, gtk3
+, atk
+, pango
+, cairo
+, gdk_pixbuf
+, glib
+, gobjectIntrospection
+, wrapGAppsHook
+, gnome3
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "neovim-gtk-${version}";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "daa84";
+    repo = "neovim-gtk";
+    rev = "30b7fc87e1e92ac2bb102fd6af6a960fc75017ac";
```

Comment:
This does not seem to be version 0.1.1. Either use `"v${version}"`, or change the version attribute to `unstable-2018-01-02`.